src/containers/RecentSubjects.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import Pusher from "pusher-js"
+import PropTypes from "prop-types"
+import apiClient from "panoptes-client/lib/api-client"
+import talkClient from 'panoptes-client/lib/talk-client';
+import Subjects from "../components/subjects"
+import { config } from "../config"
+
+class RecentSubjects extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.subjectQueue = [];
+    this.state = {
+      subjects: []
+    }
+
+    this.getSubject = this.getSubject.bind(this);
+    this.processClassification = this.processClassification.bind(this);
+  }
+
+  componentDidMount() {
+    const pusher = new Pusher('79e8e05ea522377ba6db', {encrypted: true});
+    const channel = pusher.subscribe('panoptes');
```

Comment:
We can't subscribe at a more specific (project) level? I assume not, but just want to confirm.

Reply:
I wasn't able to find out a way to do so, but it's probably worth bringing up in a Slack channel.